packages/web3-eth-abi/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "web3-eth-abi",
-	"version": "4.2.1",
+	"version": "4.3.0",
```

Comment:
there is no new feature added? so it should be 4.2.2 if there is only patch update

Reply:
Thanks for this, I thought web3-types had included a new type in web3-eth-abi but it was updated from last release. will change this, thanks